pkg/install/images/images.go

```diff
@@ -540,6 +542,10 @@ func getTemplateData(config *kubermaticv1.KubermaticConfiguration, clusterVersio
 	fakeCluster.Spec.ClusterNetwork.DNSDomain = "cluster.local"
 	fakeCluster.Spec.ClusterNetwork.KonnectivityEnabled = pointer.Bool(konnectivityEnabled)
 	fakeCluster.Spec.CNIPlugin = cniPlugin
+	fakeCluster.Spec.Features = map[string]bool{kubermaticv1.ClusterFeatureEtcdLauncher: true}
+	if etcdLauncher, exists := config.Spec.FeatureGates[kubermaticv1.ClusterFeatureEtcdLauncher]; etcdLauncher == false && exists == true {
+		fakeCluster.Spec.Features[kubermaticv1.ClusterFeatureEtcdLauncher] = false
```

Comment:
per https://docs.kubermatic.com/kubermatic/v2.22/cheat-sheets/etcd/etcd-launcher/
etcd-launcher is enabled by default but explicitly stating to disable it is possible through config feature gates, so I guess it's desired to keep `kubermatic-installer` consistent with that behaviour.